terraform/rtmp-server/nginx.conf

```diff
@@ -18,11 +18,7 @@ rtmp {
       deny publish all;
       drop_idle_publisher 10s;
       allow play all;
-      hls on;
-      hls_path /root/hls;
-      hls_sync 100ms;
-      hls_fragment 10s;
-      hls_playlist_length 60m;
+      push rtmp://10.100.80.1 app=tomesh playpath=ournetworks;
```

Comment:
Are we going to push to the clear network in this repo? or remove this line

Reply:
We need to parametrize this. Let me think how to do this in terraform. By config somehow... For this PR I'd say leave line in but comment out?

Reply:
line doesnt really do anyting because there is also the openvpn client that needs to go up.

Reply:
Right just saw this https://github.com/tomeshnet/ipfs-live-streaming/issues/21#issuecomment-400871479
Probably belongs to another PR